### PR TITLE
getting started update

### PR DESCRIPTION
**Pull request by m1ga**
update getting started guide with VSCode screenshots

### CODE REVIEW

File: docs/.vuepress/styles/index.styl [hunk 1]
```diff
@@ -181,3 +181,10 @@ a {
 .home .hero .action-button {
   color: var(--color-white) !important;
 }
+
+.page ol {
+  list-style: inherit;
+}
+ol li {
```
Comment (hansemannn) [inline docs/.vuepress/styles/index.styl:189]: Add empty line between the assignments for consistency

File: docs/.vuepress/styles/index.styl [hunk 2]
```diff
@@ -181,3 +181,10 @@ a {
...
+  margin-bottom: 20px;
```
Comment (hansemannn) [inline docs/.vuepress/styles/index.styl]: Should list margins be absolute? Please double check how they are usually set / whats the current default

**Review by hansemannn — CHANGES_REQUESTED** [review]
Two minor changes

**Comment by hansemannn** [comment]
Some merge conflicts are remaining, then we're good!

**Comment by m1ga** [comment]
already fixed :wink:

**Pull request merged**